store/tikv/client_fail_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tikv
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/pingcap/check"
+	"github.com/pingcap/failpoint"
+	"github.com/pingcap/kvproto/pkg/kvrpcpb"
+	"github.com/pingcap/tidb/config"
+	"github.com/pingcap/tidb/store/tikv/tikvrpc"
+)
+
+func (s *testClientSuite) TestPanicInRecvLoop(c *C) {
+	c.Assert(failpoint.Enable("github.com/pingcap/tidb/store/tikv/client/panicInFailPendingRequests", `return("0")`), IsNil)
```

Comment:
The failpoint name is `github.com/pingcap/tidb/store/tikv/panicInFailPendingRequests` instead of `github.com/pingcap/tidb/store/tikv/client/panicInFailPendingRequests`, all files in same directory are in the same module, which is different from `Rust`.